crmsh/utils.py

```diff
@@ -2264,4 +2265,13 @@ def get_nodeid_from_name(name):
         return res.group(1)
     else:
         return None
+
+
+def check_space_option_value(options):
+    assert isinstance(options, argparse.Namespace) is True
```

Comment:
Using `assert` in the code that is not a test is not recommend. The `assert` method is kind of special, as depending on how python is executed this methods are avoided. I would change that for a normal execption